### ISSUE TITLE
Issues running on `develop` branch

### ISSUE THREAD

**Issue by jchung00** [gnosis/dex-react#1417]
Ran `npm i` and then `npm start`, which causes this issue in logs. `localhost:8080` does not load
`(node:4681) UnhandledPromiseRejectionWarning: Error: Input buffer has corrupt header: vips_realpath: unable to form filename`

**Comment by jchung00** [comment]
Undo this commit and it works with `yarn` https://github.com/gnosis/dex-react/commit/48e5674b180b509a420dabcfad75590caa5fb283#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

**Comment by Velenir** [comment]
Can you try that script's command incrementally to see when it breaks

```sh
NODE_ENV=development npx webpack-dev-server --mode development --hot
NODE_ENV=development npx webpack-dev-server --mode development --hot --host 0.0.0.0
NODE_ENV=development npx webpack-dev-server --mode development --hot --disable-host-check
NODE_ENV=development npx webpack-dev-server --mode development --hot --host 0.0.0.0 --disable-host-check
```

**Comment by anxolin** [comment]
This was fixed in #1419
